src/LazyOperations/ExponentialMap.jl

```diff
@@ -305,17 +299,13 @@ If the direction has norm zero, the result depends on the wrapped set.
 
 If ``E = \\exp(M)⋅S``, where ``M`` is a matrix and ``S`` is a set, it
 follows that ``σ(d, E) = \\exp(M)⋅σ(\\exp(M)^T d, S)`` for any direction ``d``.
-
-We allow sparse direction vectors, but will convert them to dense vectors to be
-able to use `expmv`.
 """
 function σ(d::AbstractVector, em::ExponentialMap)
-    require(:Expokit; fun_name="σ")
+    require(SUPPORTED_EXPONENTIAL_PACKAGES; fun_name="σ")
```

Comment:
here we may as well remove this `require` call, this this anyways fails when calling `_expmat` if no backend is loaded

(same in all other functions)

Reply:
In fact we should remove these calls because otherwise you cannot add new packages from outside :+1: